pkg/backends/otlp/internal/data/metric_response.go

```diff
@@ -0,0 +1,33 @@
+package data
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	v1export "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
+	"go.uber.org/multierr"
+	"google.golang.org/protobuf/proto"
+)
+
+func ProcessMetricResponse(resp *http.Response) (dropped int64, errs error) {
+	buf, err := io.ReadAll(resp.Body)
```

Comment:
I've left it for now, since the response message unable to be passed would fail, and any errors would be pushed through proto field. 

Once I start the client and confer with the spec of the collector, I can update it then.